sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1676,6 +1677,19 @@ export class ApplicationService extends RecordDataModelService<Application> {
       );
     }
 
+    // Validates if the Program is active.
+    const program =
+      await this.educationProgramService.getActiveEducationProgram(
+        offering.educationProgram.id,
+      );
+
+    if (!program) {
+      throw new CustomNamedError(
+        "The program is not active.",
+        EDUCATION_PROGRAM_IS_NOT_ACTIVE,
```

Comment:
The custom name error is not caught and translated. Please take care of the same.

![image](https://github.com/bcgov/SIMS/assets/54600590/fd9fc395-6527-4aca-a4df-4576071fc8f1)